util/file.go

```diff
@@ -83,6 +84,7 @@ func GetPathRelativeTo(path string, basePath string) (string, error) {
 		return "", errors.WithStackTrace(err)
 	}
 
+	relPath = strings.Replace(relPath, `\`,"/", -1)
```

Comment:
Should be possible to add a helper to `test_constants_unix.go` like this:

```go
func cleanPath(path string) string {
  return strings.Replace(path, "\\", "/", -1)
}
```

And in `test_constants_windows`:

```
func cleanPath(path string) string {
  return strings.Replace(path, "/", "\\", -1)
}
```

(Perhaps a better name would be `test_helpers_unix.go` and `test_helpers_windows.go`.)

You can then wrap all the paths in the tests with a call to `cleanPath` and it should take care of the cross-platform issues...